R/S3_glance.R

```diff
@@ -0,0 +1,51 @@
+#' @importFrom generics glance
+#' @export
+generics::glance
+
+glance_lm_robust <- function(x, ...) {
+  ret <- cbind(
+    data.frame(
+      r.squared = x[["r.squared"]],
+      adj.r.squared = x[["adj.r.squared"]]
+    ),
+    if (exists("fstatistic", x)) {
```

Comment:
Instead of ternary, maybe just put an if block at the bottom.

Reply:
The reason is for the expected order, I imagine (I was mostly mimicking broom). Would you prefer an if block at the bottom with a column re-ordering?

Reply:
+1